src/schemas/validation/schema.yaml

```diff
@@ -230,6 +230,52 @@ $defs:
         type: array
         items:
           $ref: '#/$defs/parameter-or-reference'
+      additionalOperations:
+        type: object
+        additionalProperties:
+          $ref: '#/$defs/operation'
+        not:
+          anyOf:
+            - properties:
+                get:
+                  $ref: '#/$defs/operation'
+              required:
+                - get
+            - properties:
+                put:
+                  $ref: '#/$defs/operation'
+              required:
+                - put
+            - properties:
+                post:
+                  $ref: '#/$defs/operation'
+              required:
+                - post
+            - properties:
+                delete:
+                  $ref: '#/$defs/operation'
+              required:
+                - delete
+            - properties:
+                options:
+                  $ref: '#/$defs/operation'
+              required:
+                - options
+            - properties:
+                head:
+                  $ref: '#/$defs/operation'
+              required:
+                - head
+            - properties:
+                patch:
+                  $ref: '#/$defs/operation'
+              required:
+                - patch
+            - properties:
+                trace:
+                  $ref: '#/$defs/operation'
+              required:
+                - trace
```

Comment:
Does the `$ref: '#/$defs/operation'` on each `not/anyOf/*/properties/*` serve a purpose?

```yaml
not:
  anyOf:
    - properties:
        get:
          $ref: '#/$defs/operation'
      required:
        - get
```

Reads as: additionalOperations must not have a `get` property with a value that is a `#/$defs/operation` - so it _may_ have a `get` property that is not a valid operation object.
That additionalOperations object would still be invalid because the additionalOperations schema's `additionalProperties: {$ref: '#/$defs/operation'}` will fail.
I think it is simpler and validates the same instances to just have the `required` in each `not/anyOf/*` and drop the `properties`.

Reply:
not really. Thanks for calling this out. Resolving since we settled on the enum avenue to exclude those properties